jena-rdfconnection/src/main/java/org/apache/jena/rdflink/RDFLinkFuseki.java

```diff
@@ -0,0 +1,134 @@
+            .acceptHeaderQuery(acceptHeaderSPARQL)
+            .parseCheckSPARQL(false)
+            // Create object of this class.
+            .creator((b)->fusekiMaker(b));
+    }
+
+    private static RDFLinkHTTPBuilder setupCreator(RDFLinkHTTPBuilder builder) {
+        return builder.creator((b)->fusekiMaker(b));
+    }
+
+    static RDFLinkFuseki fusekiMaker(RDFLinkHTTPBuilder builder) {
+        return new RDFLinkFuseki(builder);
+    }
+
+    protected RDFLinkFuseki(RDFLinkHTTPBuilder base) {
+        this(base.txnLifecycle, base.httpClient,
+            base.destination, base.queryURL, base.updateURL, base.gspURL,
+            base.outputQuads, base.outputTriples,
+            base.acceptDataset, base.acceptGraph,
+            base.acceptSparqlResults, base.acceptSelectResult, base.acceptAskResult,
+            base.parseCheckQueries, base.parseCheckUpdates);
+    }
+
+    protected RDFLinkFuseki(Transactional txnLifecycle, HttpClient httpClient, String destination,
+                            String queryURL, String updateURL, String gspURL, RDFFormat outputQuads, RDFFormat outputTriples,
+                            String acceptDataset, String acceptGraph,
+                            String acceptSparqlResults, String acceptSelectResult, String acceptAskResult,
+                            boolean parseCheckQueries, boolean parseCheckUpdates) {
+        super(txnLifecycle, httpClient,
+              destination, queryURL, updateURL, gspURL,
+              outputQuads, outputTriples,
+              acceptDataset, acceptGraph,
+              acceptSparqlResults, acceptSelectResult, acceptAskResult, parseCheckQueries, parseCheckUpdates);
+    }
+
+    // Fuseki specific operations.
+
+//    /**
+//     * Return a {@link Model} that is proxy for a remote model in a Fuseki server. This
+//     * support the model operations of accessing statements and changing the model.
+//     * <p>
+//     * This provide low level access to the remote data. The application will be working
+//     * with and manipulating the remote model directly which may involve a significant
+//     * overhead for every {@code Model} API operation.
+//     * <p>
+//     * <b><em>Warning</em>:</b> This is <b>not</b> performant for bulk changes.
+//     * <p>
+//     * Getting the model, using {@link #fetch()}, which copies the whole model into a local
+//     * {@code Model} object, maniupulating it and putting it back with {@link #put(Model)}
+//     * provides another way to work with remote data.
+//     *
+//     * @return Graph
+//     */
+//    public Graph getGraphProxy() { return null; }
+//    public Graph getGraphProxy(String graphName) { return null; }
+//
+//    public DatasetGraph getDatasetProxy() { return null; }
+//
+//    // Or remote RDFStorage?
+//    public Stream<Triple> findStream(Node s, Node p , Node o) { return null; }
+//    public Stream<Quad> findStream(Node g, Node s, Node p , Node o) { return null; }
+
+    // Send Patch
+}
```

Comment:
This commented code is still WIP, or for future work?

Reply:
Future. We can add a Graph implementation which remotes all calls. A bit specialised.